frontend/modules/viewpoints/viewpoints.selectors.ts

```diff
@@ -18,10 +18,16 @@ import { isEmpty, orderBy, values } from 'lodash';
 import { createSelector } from 'reselect';
 import { addToGroupDictionary } from '../../helpers/colorOverrides';
 import { getTransparency, hasTransparency } from '../../helpers/colors';
-import { selectActiveIssue } from '../issues';
+import { selectActiveIssue } from '../issues'; // TODO: this should be refactored out (?)
 import { selectDefaultView } from '../model';
-import { selectActiveRisk } from '../risks';
+import { selectActiveRisk } from '../risks'; // TODO: this should be refactored out (?)
+// TODO: views/viewpoints are 2 different things
```

Comment:
remove the todo, no one ever sees it at the right time.

Just create a maintenance issue, at least that's in a kanban to consider when we do maintenance stuff.

If it already has one, then these stuff should just be description in there.